modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPSuccessTest.java

```diff
@@ -916,4 +1049,56 @@ public void testDeleteIdPTemplate() throws Exception {
                 .assertThat()
                 .statusCode(HttpStatus.SC_NO_CONTENT);
     }
+
+    /**
+     * Deletes an Identity Provider by its ID and verifies the deletion.
+     *
+     * @param idPId ID of the Identity Provider to be deleted.
+     */
+    private void deleteCreatedIdP(String idPId) {
+
+        Response response = getResponseOfDelete(IDP_API_BASE_PATH + PATH_SEPARATOR + idPId);
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_NO_CONTENT);
+
+        Response responseOfGet = getResponseOfGet(IDP_API_BASE_PATH + PATH_SEPARATOR + idPId);
+        responseOfGet.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_NOT_FOUND)
+                .body("message", equalTo("Resource not found."))
+                .body("description", equalTo("Unable to find a resource matching the provided identity " +
+                        "provider identifier " + idPId + "."));
+    }
+
+    /**
+     * Retrieves saml metadata content from the provided file.
+     * @return content of file as String
+     * @throws IOException if an error occurred while reading the file.
+     */
+    private String retrieveMetadataSamlFile(String xmlFileName) throws IOException {
```

Comment:
retrieve -> load or read

Reply:
This is fixed with 2f657bedfa3cad42fd8c65f54e3b0befbc4119e0